documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -291,7 +296,7 @@ EOF
 <7> Specify the name of the `StorageMap` CR.
 <8> For all source providers except for {virt}, you can use either the `id` _or_ the `name` parameter to specify the source VMs. +
 {virt} source provider only: You can use only the `name` parameter, not the `id.` parameter to specify the source VMs.
-<9> Specify the VMware VM MOR, {rhv-short} VM UUID or the {osp} VM UUID.
+<9> Specify the VMware VM MOR, {rhv-short} VM UUID, the {osp} VM UUID or the OVA UUID.
```

Comment:
@ahadas technically same comment applies on the VM UUID as the network but it can be more challenging to use the name since it may be not unique....